src/_guides/language/language-tour.md

```diff
@@ -158,38 +162,48 @@ mind:
-| case              | export{{bii}}     | import{{bii}}     | static{{bii}}     | while             |
-| catch             | external{{bii}}   | in                | super             | with              |
-| class             | extends           | is                | switch            | yield{{lrw}}      |
-| const             | factory{{bii}}    | library{{bii}}    | sync*{{lrw}}      | yield*{{lrw}}     |
+{% assign lrw = '&nbsp;<sup title="limited reserved word for asynchrony support" alt="limited reserved word for asynchrony support">2</sup>' %}
+{% assign lrw2 = '&nbsp;<sup title="limited reserved word" alt="limited reserved word">3</sup>' %}
+
+| abstract{{bii}}   | deferred{{bii}}   | if                | super             |
+| as{{bii}}         | do                | implements{{bii}} | switch            |
+| assert            | dynamic{{bii}}    | import{{bii}}     | sync*{{lrw}}      |
+| async{{lrw}}      | else              | in                | this              |
+| async*{{lrw}}     | enum              | is                | throw             |
+| await{{lrw}}      | export{{bii}}     | library{{bii}}    | true              |
+| break             | external{{bii}}   | new               | try               |
+| case              | extends           | null              | typedef{{bii}}    |
+| catch             | factory{{bii}}    | operator{{bii}}   | var               |
+| class             | false             | part{{bii}}       | void              |
+| const             | final             | rethrow           | while             |
+| continue          | finally           | return            | with              |
+| covariant{{lrw2}}          | for               | set{{bii}}        | yield{{lrw}}      |
+| default           | get{{bii}}        | static{{bii}}     | yield*{{lrw}}     |
 {:.table .table-striped .nowrap}
 
 <sup>1</sup> Words with the superscript **1**
-are *built-in identifiers*. Avoid using
+are **built-in identifiers**. Avoid using
 built-in identifiers as identifiers, and never use
 them for class or type names. Built-in identifiers exist to ease
 porting from JavaScript to Dart. For example,
 if some JavaScript code has a variable named `factory`, you don't have
 to rename it when you port the code to Dart.
 
 <sup>2</sup> Words with the superscript **2**
-are newer, limited reserved words related to asynchrony support
-that's being added after Dart's 1.0 release.
+are newer, limited reserved words related to **asynchrony** support
+added after Dart's 1.0 release.
 You can't use `async`, `await`, or `yield` as
 an identifier in any function body marked with `async`, `async*`, or `sync*`.
 For more information, see
 [Asynchrony support](#asynchrony-support).
 
-All other words in the keyword table are <em>reserved words</em>.
+<sup>3</sup> Words with the superscript **3**
+are newer, limited reserved words related to **type safety**
+added in 1.22 or later releases.
```

Comment:
`covariant` would simply be marked as a built-in identifier, so there should not be a need to introduce a new category of identifiers here, just use `{bii}` on covariant in the table.